x/application/client/cli/tx_stake_application_test.go

```diff
@@ -0,0 +1,103 @@
+package cli_test
+
+import (
+	"fmt"
+	"pocket/x/application/client/cli"
+	"pocket/x/application/types"
+	"testing"
+
+	errorsmod "cosmossdk.io/errors"
+	"cosmossdk.io/math"
+	sdkmath "cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/testutil"
+	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/status"
+)
+
+func TestCLI_StakeApplication(t *testing.T) {
+	net, _ := networkWithApplicationObjects(t, 2)
+	val := net.Validators[0]
+	ctx := val.ClientCtx
```

Comment:
Amazing!

Reply:
(I think this means I don't need [these test helpers anymore](https://github.com/pokt-network/poktroll/pull/42/files#diff-130d27c04c079802698072080dfccb0ce66e25a15f489ee5748b5672b91bf81eR30))